engine/src/exec-interface2.cpp

```diff
@@ -2480,8 +2480,18 @@ bool MCInterfaceStringCouldBeStack(MCStringRef p_string)
          MCStringBeginsWithCString(p_string, (const char_t *)"REVO",
                                   kMCCompareExact)))
         return true;
-    
-    // Check if it could be a script-only stack
+
+    //TODO: process BOM at the point of import into LiveCode (from URL)
```

Comment:
My thought was that if a URL keyword was presented, then the developer probably intended for the result to be decoded text (and the idea is that BOMs don't occur in usual binary data streams), so if processing the URL keyword a BOM check and decode based on that would be useful.  In any case, I'll remove the comment since that would be a separate issue to track/change.